netci.groovy

```diff
@@ -316,9 +316,17 @@ def buildArchConfiguration = ['Debug': 'x86',
                 def newJobName = "${targetGroup}_${configurationGroup.toLowerCase()}"
 
                 def newJob = job(Utilities.getFullJobName(project, newJobName, isPR)) {
-                    // On Windows we use the packer to put together everything. On *nix we use tar
-                    steps {
-                        batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" x86 && build.cmd -${configurationGroup} -framework:${targetGroup}")
+                    if (targetGroup == 'netfx') {
```

Comment:
Will wait to @joperezr to see if we want to start building tests for uap configurations. If so, then we can delete the condition and just add the build-tests.cmd command for all target groups.